test/witan/app/forecast_test.clj

```diff
@@ -0,0 +1,31 @@
+(ns witan.app.forecast-test
```

Comment:
I'm a bit worried about use of `with-redefs`, which is known to cause trouble (particularly when threads are involved), I'd prefer to architect the app using stuart sierra component instead which would allow injecting mock/stubs instead, but that's a bigger discussion and the use here is well isolated.